src/games/strategy/ui/ZoomableImage.java

```diff
@@ -37,10 +37,8 @@ private void ensureImageLoaded(final Image anImage) {
     tracker.addImage(anImage, 1);
     try {
       tracker.waitForAll(1);
-    } catch (final InterruptedException ie) {
-      ie.printStackTrace();
-      System.err.println("<<<<<<<<<<<<<<<<<<<<<<>trying again to load Image");
-      ensureImageLoaded(anImage);
```

Comment:
No. It is a retry when there was an interrupted exception. An interrupted exception would happen if someone "ctrl+c" / killed the application. We should only be doing cleanup if there is any to be done, and exiting.